src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/DataSetFilesControllerTests.cs

```diff
@@ -95,6 +95,51 @@ await TestApp.AddTestData<ContentDbContext>(context =>
                 AssertResultsForExpectedReleaseFiles(publication1Release1Version1Files, pagedResult.Results);
             }
 
+            [Fact]
+            public async Task FilterByGeographicLevel_Success()
+            {
+                var (publication1, publication2) = _fixture
+                    .DefaultPublication()
+                    // Publications each have a published release version
+                    .WithReleases(_fixture.DefaultRelease(publishedVersions: 1)
+                        .Generate(1))
+                    .WithTheme(_fixture.DefaultTheme())
+                    .GenerateTuple2();
+
+                var publication1Release1Version1Files = _fixture.DefaultReleaseFile()
+                    .WithReleaseVersion(publication1.ReleaseVersions[0])
+                    .WithFiles(_fixture.DefaultFile(FileType.Data)
+                    .WithDataSetFileMeta(_fixture.DefaultDataSetFileMeta()
+                        .WithGeographicLevels(
+                            [GeographicLevel.Country, GeographicLevel.LocalAuthority, GeographicLevel.Institution]))
+                    .WithDataSetFileGeographicLevels(
+                        [GeographicLevel.Country, GeographicLevel.LocalAuthority, GeographicLevel.Institution])
```

Comment:
I think this is an example of where it can get a bit confusing keeping geographic levels in `DataSetFileMeta` when it's now also in the collection of `DataSetFileGeographicLevel`. Presumably only one set of levels needs to be set up here?

Reply:
It uses DataSetFileVersionGeographicLevel to filter in the db query, but then uses the DataSetFileMeta to populate the response.

This will get sorted as part of EES-5666 which I'll be picking up after the filter hierarchies work